helmfile/charts/notify-documentation/templates/deployment.yaml

```diff
@@ -3,22 +3,22 @@ kind: Deployment
 metadata:
   name: {{ include "notify-documentation.fullname" . }}
   labels:
-    {{- include "notify-documentation.labels" . | nindent 4 }}
+    app: {{ include "notify-documentation.name" . }}
```

Comment:
Being pedantic again, but this should probably be in camel case like everything else... notifyDocumentation... or just documentation really since we don't say notifyApi or notifyAdmin in the others.